packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -314,6 +330,7 @@ export class AtomicSearchInterface {
       i18n: this.i18n,
       store: this.store,
       interfaceElement: this.host,
+      interfaceComponent: this,
```

Comment:
This adds a reference to the search interface as a component, instead of only as an HTMLElement.

Reply:
Why not adding the asset paths to the bindings?

Reply:
> Why not adding the asset paths to the bindings?

That seems too narrow, though ?

As in, I can see being able to reference the search interface component (and all it's props) as more generically useful for a lot more components in the future, than just the asset path themselves.

Reply:
I'm just worried that would be too "wide". In the future we might have other types of parent interface component with not all the same props/methods.

You could technically just make the bindings the component and have all properties be accessible

Reply:
Good point. It's indeed not generic enough. 

Alright, I can change it to assets path...

What about adding it only to `<atomic-icon asset-path="./foo">` though ? 

You would need to set it as a prop on every icon you use in your interface, but it would be more direct/scoped.

Reply:
Alternatively, it could be in the store.

> What about adding it only to `<atomic-icon asset-path="./foo">` though ?
> 
> You would need to set it as a prop on every icon you use in your interface, but it would be more direct/scoped.

I don't expect someone to use `atomic-icon` outside of a search interface, I think it's more convenient and looks less cluttered to read bindings from `atomic-icon`.

Reply:
Moved to store